### PR TITLE
UI testing

### PR DESCRIPTION
**Pull request by danacebryantalfonso**
my latest activity

### CODE REVIEW

File: SocialIntegration/app/src/androidTest/java/com/exercise/socialintegration/SocIntTest.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    )
+
+    @Test
+    fun test_login_error_access_denied()
+    {
+        onView(withId(R.id.et_signin_username)).perform(typeText("test123"))
+        onView(isRoot()).perform(waitFor(2000))
+        onView(withId(R.id.et_signin_password)).perform(scrollTo(), typeText("test1"))
+        onView(isRoot()).perform(waitFor(5000))
+        onView(withId(R.id.btn_mlogin)).perform(closeSoftKeyboard(),scrollTo(), click())
```
Comment (rexendy-lig) [inline SocialIntegration/app/src/androidTest/java/com/exercise/socialintegration/SocIntTest.kt:36]: after you perform a certain function, pls assert something that proves it denies or login successfully
pls apply to other test cases also

File: SocialIntegration/app/src/main/java/com/exercise/socialintegration/adapter/UserViewHolder.kt [hunk 2]
```diff
@@ -1,12 +1,66 @@
...
+            }
+
+            this.tvPassword.text = txtPassword
+            this.tvUsername.text = txtUsername
+
+            /*this.editButton.setOnClickListener {
+            val intent: Intent = Intent(context, EditTodoActivity::class.java)
+            intent.putExtra("id", model.id)
+            this.context.startActivity(intent)
+        }*/
```
Comment (rexendy-lig) [inline SocialIntegration/app/src/main/java/com/exercise/socialintegration/adapter/UserViewHolder.kt:42]: remove if not needed

File: SocialIntegration/app/src/main/java/com/exercise/socialintegration/ui/signin/SignInFragment.kt [hunk 3]
```diff
@@ -40,36 +43,34 @@ class SignInFragment : Fragment() {
...
-                .equalTo("username", txtUsername.getText().toString())
-                .equalTo("password", txtPassword.getText().toString())
+                .equalTo("username", txtUsername.text.toString())
+                .equalTo("password", txtPassword.text.toString())
                 .count()
             if(userId.toInt() > 0) {
-                sv.scrollTo(0,0)
                 Toast.makeText(view.context,"Logging in...",Toast.LENGTH_LONG).show()
                 showHome()
             }else{
```
Comment (rexendy-lig) [inline SocialIntegration/app/src/main/java/com/exercise/socialintegration/ui/signin/SignInFragment.kt:62]: add space after if and {
ex
if () {
} else {
}

**Review by rexendy-lig — CHANGES_REQUESTED** [review]

File: SocialIntegration/app/src/androidTest/java/com/exercise/socialintegration/SocIntTest.kt (hunk 1 above)
Comment (danacebryantalfonso) [reply SocialIntegration/app/src/androidTest/java/com/exercise/socialintegration/SocIntTest.kt]: done in b4486f9909e16c5bda5b8c2852ca12b612ca1c9c

File: SocialIntegration/app/src/main/java/com/exercise/socialintegration/adapter/UserViewHolder.kt (hunk 2 above)
Comment (danacebryantalfonso) [reply SocialIntegration/app/src/main/java/com/exercise/socialintegration/adapter/UserViewHolder.kt]: done in b4486f9909e16c5bda5b8c2852ca12b612ca1c9c

File: SocialIntegration/app/src/main/java/com/exercise/socialintegration/ui/signin/SignInFragment.kt (hunk 3 above)
Comment (danacebryantalfonso) [reply SocialIntegration/app/src/main/java/com/exercise/socialintegration/ui/signin/SignInFragment.kt]: done in b4486f9909e16c5bda5b8c2852ca12b612ca1c9c

Comment (danacebryantalfonso) [reply SocialIntegration/app/src/main/java/com/exercise/socialintegration/ui/signin/SignInFragment.kt]: revised in 51259d1a67d2cd5e7b9f436730a9c698e7684308

**Review by rexendy-lig — APPROVED** [review]

**Pull request merged**